### PR TITLE
Document the usage of Redis

### PR DESCRIPTION
**Pull request by stefnats**
After being stuck on this for an hour, i finally found the solution which i documented here.

### CODE REVIEW

**Comment by nunomaduro** [comment]
Would be cool if you could create a component with this, and update the docs as component too. Here is an example: https://github.com/laravel-zero/framework/pull/351/files.

**Comment by stefnats** [comment]
Hey Nuno, will do as soon as i have time :) Thanks for the feedback.

**Comment by nunomaduro** [comment]
Thank you for starting this initiative!

File: source/docs/database.md [hunk 1]
```diff
@@ -29,3 +29,15 @@ $users = DB::table('users')->get();
 
 Laravel [Database Migrations](https://laravel.com/docs/migrations), [database factories](https://laravel.com/docs/5.8/database-testing#writing-factories), and [Database Seeding](https://laravel.com/docs/seeding) features are also included.
 
+If you want to use Redis, include illuminate/redis in your composer file and add the RedisServiceProvider in your `config/app.php`
```
Comment (owenvoke) [inline source/docs/database.md:32]:
```suggestion
If you want to use Redis, include `illuminate/redis` in your Composer file and add the `RedisServiceProvider` in your `config/app.php`.
```

**Review by owenvoke — COMMENTED** [review]
Just a small suggestion for changes to the documentation wording for consistency. 👍

Edit: Just seen that Nuno mentioned it could be added as a component, so this might not really be relevant.

**Pull request closed without merging**